docs/introduction/framework-extensibility.md

```diff
@@ -0,0 +1,31 @@
+# Framework Extensibility
+
+This article summarizes current possibilities of the framework extension,
+provides a list of customizations that are not achievable now but are planned to be enabled soon, 
+as well as a list of customizations that are not (and will not be) possible at all.
+
+## What Is Achievable Easily
```

Comment:
We do not capitalize second level headings in the rest of the docs. Please remove the capitals.